### DISCUSSION TITLE
why's my windows antiviurs showing detecting a malware

### DISCUSSION THREAD

**Discussion by devhindo** [goreleaser/goreleaser#4757] [Q&A] [answered]
![image](https://github.com/goreleaser/goreleaser/assets/66886118/330ba5c6-7d98-43fb-898e-6b0360c39668)

**Comment by caarlos0** [comment]
probably a false positive... on winget's repo goreleaser itself gets flagged constantly... 

Not sure if we can mitigate that in any way...

**Reply by devhindo** [reply]
what got my attention is the release before that ([1.0.0](https://github.com/devhindo/x/releases/tag/v1.0.0)) doesn't get flagged. But the latest ([1.1.0](https://github.com/devhindo/x/releases/tag/v1.1.0)) gets flagged even though my code doesn't change a lot only modified some basic functionality and I used the same version of goreleaser in both builds so I can't find the reason why this release gets flagged by Windows

**Reply by dehanj** [reply, marked as answer]
I don't think this has to do with GoReleaser, but with Go. It is even mentioned in their own FAQ, see here: https://go.dev/doc/faq#virus

I experienced similar issues when releasing an Go application last fall, found out that the version of Go made a lot of difference if it got flagged as a malware or not.
